apps/cookbook/src/app/showcase/button-showcase/button-showcase.component.ts

```diff
@@ -42,7 +40,8 @@ export class ButtonShowcaseComponent {
     },
   ];
 
-  onChange(value) {
-    this.themeColor = value;
+  scrollTo(target: Element) {
+    target.scrollIntoView({ behavior: 'smooth' });
+    return false;
```

Comment:
Would have been nice if we could just use anchors and `scroll-behavior: smooth;`. That would improve accessibility. But I guess that's not possible due to Angular's routing?

Anyway, it's not a big deal since the cookbook doesn't have to meet the accessibility requirements - at least not to a certain degree, I guess 😊

Reply:
Yep excactly - it's related to Angulars `HashLocationStrategy`, which currently prevents us from using regular html anchors. I've looked into it earlier, but it's a can of whoop-ass 🙈 😂 

Anyhoo - I would also say this is unrelated to/ out of scope of the PR at hand, as this scrolls the same way we do everywhere else 🤷🏻